src/Themes/ui/StyleEditorModal.tsx

```diff
@@ -106,8 +106,10 @@ export function StyleEditorModal(props: IProps): React.ReactElement {
         <StyleField<"fontSize">
           name="Font Size"
           type="number"
-          value={customStyle.fontSize}
-          onChange={(value, error) => update({ ...customStyle, fontSize: Math.max(5, Number(value) ?? 8) }, error)}
+          value={customStyle.fontSize * (16 / 14)}
+          onChange={(value, error) =>
+            update({ ...customStyle, fontSize: Math.max(5, (Number(value) ?? 8) * (14 / 16)) }, error)
```

Comment:
What's up with this 14/16 adjustment? It's really weird. I'm pretty sure what you actually want to do is `+ "px"`

Reply:
That makes sense. I added this as a comment to the source, it's weird enough that we shouldn't lose track of it.